mistralrs-core/src/pipeline/mod.rs

```diff
@@ -219,6 +219,9 @@ pub trait Pipeline: Send + Sync {
     fn get_max_seq_len(&self) -> usize;
     fn is_xlora(&self) -> bool;
     fn has_no_kv_cache(&self) -> bool;
+    fn get_prefill_chunk_size(&self) -> usize {
+        512
+    }
```

Comment:
Should this be configurable?

This value should be as large as possible for performance, under the constraint that it doesn't OOM the system.